### DISCUSSION TITLE
No search capability when viewing code.

### DISCUSSION THREAD

**Discussion by takov751** [community/community#28364] [Mobile] [answered]
In the app when code opened for view can't find any way to search in file.
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.75.0 (74400986)
Device: iPhone 11 (iOS 16.0)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @takov751 thanks for the request. I passed this onto to the team for prioritization!

**Comment by luion22tikar** [comment]


**Reply by Salel46207** [reply]
> Hi Hello, how are you? Thank you🌹🌹🌹

salim46207@gmail.com

**Reply by Salel46207** [reply]
salim46207@gmail.com
